src/feeds/account-news.feed.ts

```diff
@@ -1,21 +1,20 @@
 import { Expose, plainToClassFromExist } from 'class-transformer';
```

Comment:
All the filenames should respect containing classname and vice-versa. So filename in this case should be `news.feed.ts`